show/plugins/pbh.py

```diff
@@ -0,0 +1,454 @@
+                    'is-leaf-list': False,
+                    'is-mandatory': True,
+                    'group': ''}
+            ),
+            format_attr_value(
+                entry,
+                {
+                    'name': 'packet_action',
+                    'description': 'Configures packet action for this rule',
+                    'is-leaf-list': False,
+                    'is-mandatory': False,
+                    'group': ''
+                }
+            ),
+            format_attr_value(
+                entry,
+                {
+                    'name': 'flow_counter',
+                    'description': 'Configures packet action for this rule',
+                    'is-leaf-list': False,
+                    'is-mandatory': False,
+                    'group': ''
+                }
+            ),
+        ]
+
+        body.append(row)
+
+    # sorted by 'Priority'
+    body_sorted = sorted(body, key=lambda e: int(e[2]), reverse=True)
+    click.echo(tabulate.tabulate(body_sorted, header, numalign="left"))
+
+
+@PBH.group(name="table",
+           cls=clicommon.AliasedGroup,
+           invoke_without_command=True)
+@clicommon.pass_db
+def PBH_TABLE(db):
+    """  Show the PBH table configuration """
+
+    header = [
+        "NAME",
+        "INTERFACE",
+        "DESCRIPTION",
+    ]
+
+    body = []
+
+    table = db.cfgdb.get_table("PBH_TABLE")
+    for key in natsort.natsorted(table):
+        entry = table[key]
+        if not isinstance(key, tuple):
+            key = (key,)
+
+        row = [*key] + [
+            format_attr_value(
+                entry,
+                {
+                    'name': 'interface_list',
+                    'description': 'Interfaces to which this table is applied',
+                    'is-leaf-list': True,
+                    'is-mandatory': False,
+                    'group': ''
+                }
+            ),
+            format_attr_value(
+                entry,
+                {
+                    'name': 'description',
+                    'description': 'The description of this table',
+                    'is-leaf-list': False,
+                    'is-mandatory': True,
+                    'group': ''
+                }
+            ),
+        ]
+
+        body.append(row)
+
+    click.echo(tabulate.tabulate(body, header, numalign="left"))
+
+
+@PBH.group(name="statistics",
+           cls=clicommon.AliasedGroup,
+           invoke_without_command=True)
+@clicommon.pass_db
+def PBH_STATISTICS(db):
+    """  Show the PBH counters """
+
+    header = [
+        "TABLE",
+        "RULE",
+        "RX PACKETS COUNT",
+        "RX BYTES COUNT",
+    ]
+
+    body = []
+
+    pbh_rules = db.cfgdb.get_table("PBH_RULE")
+    pbh_counters = read_pbh_counters(pbh_rules)
+    saved_pbh_counters = read_saved_pbh_counters()
+
+    for key in pbh_rules:
+        # if counters value 0 or not exists should we display it?
```

Comment:
if value of `FLOW_COUNTER` for PBH RULE is `DISABLED` we should NOT display counters enty

Reply:
consider the case if the user has disabled the `FLOW_COUNTER` value and this value exist in `saved_pbh_counters`

Reply:
WE SHOULD display counters even if value is 0